Sources/SwiftGit2/SwiftGit2.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SwiftGit2.swift
+//
+//
+//  Created by Mathijs Bernson on 01/03/2024.
+//
+
+import Foundation
+import Clibgit2
+
+public enum SwiftGit2 {
```

Comment:
I've discovered a problem with this approach to namespacing. It makes it no longer possible to reference a type in SwiftGit2 using its fully qualified name (such as `SwiftGit2.Tag`).

I will change these into freestanding functions to avoid that problem.